### PR TITLE
docs(one_dashboard_json): addition of a multipage dashboard example and other corrections

### PR DESCRIPTION
**Pull request by pranav-new-relic**
# Description

This PR addresses [NR-108017](https://issues.newrelic.com/browse/NR-108017)
- Addition of an example to demonstrate creating a dashboard with multiple pages, using the resource **newrelic_one_dashboard_json**
- A tiny correction to a link in **newrelic_notification_channel**. Despite the presence of the right Slack sample URL in the following code snippet (with \<userId>/\<channelId>) 

https://github.com/newrelic/terraform-provider-newrelic/blob/77bab1b281a8ad5882956b39d5a1644e16d37028/website/docs/r/notification_channel.html.markdown?plain=1#L80

, the link is [seen to be rendered incorrect](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/notification_channel#SLACK) in the docs of **newrelic_notification_channel**. An escape character has been added to the symbols to help render the right link (this is found to be working in the Terraform Doc Preview Tool.)

image

**Fixes** #2180 #2192 

## Type of change
- [x] This change requires a documentation update

## Checklist:
Please delete options that are not relevant.
- [x] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: website/docs/r/notification_channel.html.markdown [hunk 1]
```diff
@@ -77,7 +77,7 @@ Each notification channel type supports a specific set of arguments for the `pro
   * `email` - (Required) Specifies the user email for integrating with Pagerduty.
   * `customDetails` - (Optional) Free text that *replaces* the content of the alert.
 * `SLACK`
-  * `channelId` - (Required) Specifies the Slack channel id. This can be found in slack browser via the url. Example - https://app.slack.com/client/<UserId>/<ChannelId>.
+  * `channelId` - (Required) Specifies the Slack channel id. This can be found in slack browser via the url. Example - https://app.slack.com/client/\<UserId>/\<ChannelId>.
```
Comment (pranav-new-relic) [inline website/docs/r/notification_channel.html.markdown:80]: For the reviewer: Kindly refer to the description in the PR explaining why these changes were made.

File: website/docs/r/one_dashboard_json.html.markdown [hunk 2]
```diff
@@ -107,6 +107,396 @@ resource "newrelic_one_dashboard_json" "bar" {
   "variables": []
 }
 ```
+### Dashboard With Multiple Pages
+
+The following example demonstrates creating a dashboard with multiple pages, with each page comprising a couple of widgets.
+
+`dashboard.json`
```
Comment (kidk) [inline website/docs/r/one_dashboard_json.html.markdown:114]: Can you include what the resource would look like?

File: website/docs/r/one_dashboard_json.html.markdown [hunk 3]
```diff
@@ -107,6 +107,396 @@ resource "newrelic_one_dashboard_json" "bar" {
...
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [
+                  {Account-IDs}
```
Comment (kidk) [inline website/docs/r/one_dashboard_json.html.markdown:147]: Can you change this to `account_id` in all examples? That way it's persistent with all examples and Terraform.

**Review by kidk — CHANGES_REQUESTED** [review]

File: website/docs/r/one_dashboard_json.html.markdown (hunk 2 above)
Comment (pranav-new-relic) [reply website/docs/r/one_dashboard_json.html.markdown:119]: [Added a HCL block](https://github.com/newrelic/terraform-provider-newrelic/pull/2347/files#diff-3c4c21011e01919758a5262f44acd65c06b6c4f35b259806978a2e637d342790R113-R117) in the most recent commit.

File: website/docs/r/one_dashboard_json.html.markdown (hunk 3 above)
Comment (pranav-new-relic) [reply website/docs/r/one_dashboard_json.html.markdown]: Replaced all instances of `{Account-IDs}` with `account_id` in the examples.

File: website/docs/r/one_dashboard_json.html.markdown [hunk 4]
```diff
@@ -107,6 +107,400 @@ resource "newrelic_one_dashboard_json" "bar" {
   "variables": []
 }
 ```
+### Dashboard With Multiple Pages
+
+The following example demonstrates creating a dashboard with multiple pages, with each page comprising a couple of widgets.
+```hcl
+resource "newrelic_one_dashboard_json" "foo" {
+   json = file("dashboard.json")
```
Comment (kidk) [inline website/docs/r/one_dashboard_json.html.markdown:115]: Can we use templatefile and replace the account_id variables?

Comment (pranav-new-relic) [reply website/docs/r/one_dashboard_json.html.markdown]: Changed this in the latest commit https://github.com/newrelic/terraform-provider-newrelic/pull/2347/commits/c290a00a888af4080aadd108f8fcd3e6cfe9d0e6.

`account_id` is now passed from the Terraform configuration into the template file.

**Review by kidk — APPROVED** [review]
